komsic/XChanger
# #169800688 Connect to Server

#### What does this PR do?

Connect the client-side to the server

#### Description of Task to be completed?

- connect the app to fetch the currency rates from the backend
- add a loading state to the app
- make the currency reducer compatible with data fetching
 

#### How should this be manually tested?

- clone the repo
- cd into it
- run `yarn` on cmd
- run `yarn start` to see the implemented repo

#### What are the relevant pivotal tracker stories?

[#169800688](https://www.pivotaltracker.com/story/show/169800688)
